guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -65,18 +62,16 @@ ifdef::katello,satellite,orcharhino[]
 Use an activation key to simplify specifying the environments.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
 endif::[]
+ifeval::["{context}" == "load-balancing"]
 +
-To enter a password as command line argument, use `username:password` syntax.
-Keep in mind this can save the password in the shell history.
+Include `{ "smart_proxy_id": __{SmartProxy}_ID__ }`.
+You can use the ID of any {SmartProxyServer} that you configured for host registration load balancing.
+{Project} will apply the load balancer to the registration command automatically.
 +
-For more information about registration see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering a Host to {ProjectName}] in _{ManagingHostsDocTitle}_.
-. Log in to the host you want register and run the previously generated command.
-. Update subscription manager configuration for `rhsm.baseurl` and `server.hostname`:
+Include `{ "force": true }` to register a host that has been previously registered to a {SmartProxyServer}.
+endif::[]
 +
-[options="nowrap" subs="+quotes,attributes"]
-----
-# subscription-manager config \
---rhsm.baseurl=https://loadbalancer.example.com/pulp/content \
---server.hostname=loadbalancer.example.com
-----
+To enter a password as command line argument, use `username:password` syntax.
```

Comment:
Nit: Users can generate _Personal Access Token_ for the API with expiration date. It's in _My Account > Personal Access token_. It's good alternative to the login:password method